recipes/r-maybe/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = '1.0.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-maybe
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/maybe_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/maybe/maybe_{{ version }}.tar.gz
+  sha256: c9818d3c5c1a857b51fe1a58d6c94eb4bc6674fcc26a11907d75efe795223cf8
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip  # [win]
+
+  host:
+    - r-base
+    - r-magrittr
+
```

Comment:
```suggestion
  - cross-r-base {{ r_base }}    # [build_platform != target_platform]
```

Reply:
done on my end.

Reply:
Removed this as it conflicted with being noarch